### DISCUSSION TITLE
could not authenticate web token with miner

### DISCUSSION THREAD

**Discussion by bugy855** [UpstreamData/pyasic#115] [Q&A]
Hello, I was wonder what causes this error and what options I have to remediate it, I could not find any login options I can provide when creating a miner so I am at a loss of what the proper way of handling this is 
![image](https://github.com/UpstreamData/pyasic/assets/48725244/ecff98bd-25dd-4256-95f6-17f08c02e220)

**Comment by b-rowan** [comment]
You have a few options here - 

1. Set `miner.web.pwd` to your desired password,
2. Set `pyasic.settings.set("default_vnish_web_password", "your_pwd")` to set the password globally.

**Reply by bugy855** [reply]
setting globally won't work for my case but option #1 may, I will give that a try, thank you for the help! Is there some documentation that covers this that I just haven't found yet?
